docs/en/index.md

```diff
@@ -86,14 +86,14 @@ All filesystems are automatically mounted to the mount manager service for easy
 class FileUploadPresenter extends BasePresenter
 {
     /**
-     * @var League\Flysystem\MountManager
+     * @var \League\Flysystem\MountManager
```

Comment:
Of course, but there is no **use** in the example. In examples it is better to add leading \, so anyone can copy and paste without pain.